### DISCUSSION TITLE
Refine Promotion/Demotion Rules for EG Member

### DISCUSSION THREAD

**Discussion by MaximilianHauer** [catenax-eV/catenax-ev.github.io#129] [General]
Currently due to Catena-X NEXT we experience the situation that the established Expertgroup is getting enhanced by person that where not included during the creation of the EG.
Basically these person skipped the registration & approval phase during the EG creation. 
For this we would need a clearly defined ruleset

**Open Topics:**

1. [Expert Group Member](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/expert-group#expert-group-member) should be enhanced by the definition of a Core & Review Member . Is a Review member allowed to work in the expert group ?  
3. How can an interested Person request to participate in an existing EG. How does the person find out what is existing. 
4. who is responsible to accept new members and assigns their role 
5. What roles can new members receive in the beginning are they able to become core member directly? is there a transition phase form review to core member ? (committer example ? )

**Comment by MaximilianHauer** [comment]
@stephanbcbauer / @Grand-Thibault  fyi

**Comment by MaximilianHauer** [comment]
First idea drafted on Friday Meeting 

![image](https://github.com/user-attachments/assets/986df88c-7406-4ebd-b9ae-d734a052fcb3)

**Reply by Grand-Thibault** [reply]
I Like the picture! What has to be added in restrictions to the Core Team (on member per company) and some demotion process or generally speaking a process to have a regular change of people in the Core Team. Every (new) association member should be able to join the Core Team on a defined time horizon. They should not have to wait until a seat gets vacant.

**Reply by MaximilianHauer** [reply]
> restrictions to the Core Team (on member per company)

i thought about it but this is implicitly solved as you can become only core member if your company is not providing a core member already , the second member of a company would be automtically review member. therefor you can only vote once anyway. 

> association member should be able to join the Core Team on a defined time horizon.

we discussed this too and i find it better to provide the responsibility of defining this horizon in the EG not ruling this. May it be that there is a need that this review member phase is shorter for one person as we need more core member as 1-2 will leave on their will. may it be that we are overstaffed and the people are active and there is no clear "value" in replacing . 

a replacing should be always a benefit for the EG in regards of value.

**Comment by MaximilianHauer** [comment]
in the last retro meeting it was brought up that there should be furthermore a chance to demote from core to review members if they are not contributing in an expected manner.
a scenario for this case would be people that affect certain deliveries with their vote but wont contribute into the work before at all.
